<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyzer</class>
 <widget class="QMainWindow" name="Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{font: 75 14pt &quot;Candara&quot;;}
QLabel{font: 75 12pt &quot;Candara&quot;;}
QPushButton{
	border: None;
	padding:5px;
	color: rgb(255, 255, 255);
	
	
	background-color: rgb(53, 123, 208);
	border-radius: 7px; 
}

QPushButton:hover{
	

	background-color: rgb(22, 52, 86);
	
	
	
}

QPushButton:pressed{

	
	background-color: rgb(8, 8, 8);
}
	</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 123, 208);
color: rgb(255, 255, 255);
font: 75 22pt &quot;Candara&quot;;</string>
          </property>
          <property name="text">
           <string>COVID-19 DATA ANLYZER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Chart Dispaly Area</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="file_path">
          <property name="text">
           <string>file name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="file_open">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>--select column--</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="col_list"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="MplWidget" name="MplWidget" native="true"/>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="user_output">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 87 15pt &quot;Arial&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>output</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QPushButton" name="reset_index_btn">
        <property name="text">
         <string>reset index</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="drop_null_btn">
        <property name="text">
         <string>drop null rows</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="std_dv_btn">
        <property name="text">
         <string>standard deviation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="drop_columns_btn">
        <property name="text">
         <string>drop column</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="empty_values_btn">
        <property name="text">
         <string>check empty value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="mean_btn">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>mean</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="rename_col_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton{font: 75 16pt &quot;Candara&quot;;
}

QPushButton{
	border: None;
	padding:5px;
	color: rgb(255, 255, 255);
	
	
	background-color: rgb(53, 123, 208);
	border-radius: 7px; 
}

QPushButton:hover{
	

	background-color: rgb(22, 52, 86);
	
	
	
}

QPushButton:pressed{

	
	background-color: rgb(8, 8, 8);
}
	</string>
        </property>
        <property name="text">
         <string>rename column</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="check_punct_btn">
        <property name="text">
         <string>check punctuations</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="max_btn">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="median_btn">
        <property name="text">
         <string>median</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="hist_btn">
        <property name="text">
         <string>histogram</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="data_shape_btn">
        <property name="text">
         <string>data shape</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="columns_btn">
        <property name="text">
         <string>columns</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="min_btn">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pie_chart_btn">
        <property name="text">
         <string>pie chart</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="drop_dup_btn">
        <property name="text">
         <string>drop duplicate rows</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="check_dup_btn">
        <property name="text">
         <string>check duplicates</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="clear_output_btn">
        <property name="text">
         <string>clear output</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="new_col_name">
      <property name="styleSheet">
       <string notr="true">QLineEdit:{font: 75 18pt &quot;Candara&quot;;}
background-color: rgb(85, 255, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>--enter new column name--</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
